Terminal.Gui/ConsoleDrivers/WindowsDriver.cs

```diff
@@ -1695,7 +1696,13 @@ private MouseFlags ProcessButtonClick (WindowsConsole.MouseEventRecord mouseEven
 
     private async Task ProcessButtonDoubleClickedAsync ()
     {
-        await Task.Delay (300);
+        // Only delay if there's not a view wanting continuous button presses
+        if (Application.WantContinuousButtonPressedView is null)
```

Comment:
This isn't necessary because this async method is only called if the `_isButtonDoubleClicked or _isOneFingerDoubleClicked` is `true`. So, even a view which `WantContinuousButtonPressed` is `true` may want a `ButtonTripleClicked` event. Another detail, for a click, double click or triple click event occurs, it means that were also button released each time, which will generate the clicked events.
Am I missing something?